docs/containers/reference.md

```diff
@@ -392,6 +392,50 @@ Here are all properties available for configuring `docker-run` task. All propert
 | `configureSsl` | Whether to configure ASP.NET Core SSL certificates and other settings to enable SSL on the service in the container. |
 | `enableDebugging` | Whether to enable the started container for debugging. This will infer additional volume mappings and other options necessary for debugging. |
 
+## Docker Compose task
```

Comment:
An example value would be helpful in here since the task must be manually constructed (it isn't scaffolded automatically by anything).